docs/classic-vs-vt.md

```diff
@@ -0,0 +1,81 @@
+
+A limited subset of Windows Console APIs are still necessary to establish the initial environment as the Windows platform still differs from others in process, signal, device, and encoding handling:
+
+- The standard handles to a process will still be controlled with **[GetStdHandle](getstdhandle.md)** and **[SetStdHandle](setsetdhandle.md)**.
+- Configuration of the console modes on a handle to opt in to Virtual Terminal Sequence support will be handled with **[GetConsoleMode](getconsolemode.md)** and **[SetConsoleMode](setconsolemode.md)**.
+- Declaration of code page or UTF-8 support is conducted with [**SetConsoleOutputCP**](setconsoleoutputcp.md) and [**SetConsoleCP**](setconsolecp.md) methods.
+- Some level of overall process management may be required with the [**AllocConsole**](allocconsole.md), [**AttachConsole**](attachconsole.md) and [**FreeConsole**](freeconsole.md) to join or leave a console device session.
+- Signals and signal handling will continue to be conducted with [**SetConsoleCtrlHandler**](setconsolectrlhandler.md), [**HandlerRoutine**](handlerroutine.md), and [**GenerateConsoleCtrlEvent**](generateconsolectrlevent.md).
+- Communication with the console device handles can be conducted with [**WriteConsole**](writeconsole.md), [**ReadConsole**](readconsole.md), [**WriteFile**](https://msdn.microsoft.com/library/windows/desktop/aa365747) and [**ReadFile**](https://msdn.microsoft.com/library/windows/desktop/aa365467).
+    - These may also be leveraged through programming language runtimes in the forms of:
+        - C Runtime (CRT): [Stream I/O](https://docs.microsoft.com/cpp/c-runtime-library/stream-i-o) like **printf**, **scanf**, **putc**, **getc**, or [other levels of I/O functions](https://docs.microsoft.com/cpp/c-runtime-library/input-and-output).
+        - C++ Standard Library (STL): [iostream](https://docs.microsoft.com/cpp/standard-library/iostream) like **cout** and **cin**.
+        - .NET Runtime: [System.Console](https://docs.microsoft.com/dotnet/api/system.console) like **Console.WriteLine**.
+
+There are no plans to remove the Windows console APIs from the platform. On the contrary, the Windows console host has provided the [pseudoconsole](pseudoconsoles.md) technology to translate existing Windows command-line application calls into virtual terminal sequences and forward them to another hosting environment remotely or across platforms. This translation is not perfect and requires the console host window to maintain a simulated environment of what Windows would have displayed to the user and project a replica of that to the pseudoconsole host. Any console API call without an equivalent is operated within the simulated environment to serve the needs of the legacy command-line client application and only the effects are propagated onto the final host.

```

Comment:
Soften wording. _all_ APIs hit the simulated environment, so maybe we should be clear about that?

Until we start turning `SetConsoleCursorPosition` into to `printf("\e[%u,%uH", ...)`, that is...

Reply:
I did change it to "all apis hit simulated environment" but I don't understand what you want "softened". Can you give an example?